components/src/components/molecules/Dropdown/Dropdown.tsx

```diff
@@ -540,6 +550,7 @@ export const Dropdown = ({
           return React.cloneElement(child as any, {
             ...buttonProps,
             zindex: '10',
+            pressed: isOpen,
```

Comment:
not all custom elements will have a pressed attribute, this shows a warning in unit tests

Reply:
I switched it to passing a string when isOpen is true since strings are valid values for injected properties.